src/engine/effects/engineeffectchain.cpp

```diff
@@ -128,50 +128,42 @@ bool EngineEffectChain::processEffectsRequest(const EffectsRequest& message,
     return true;
 }
 
-bool EngineEffectChain::enableForGroup(const QString& group) {
-    GroupStatus& status = getGroupStatus(group);
+bool EngineEffectChain::enableForChannel(const ChannelHandle& group) {
+    ChannelStatus& status = getChannelStatus(group);
     if (status.enable_state != EffectProcessor::ENABLED) {
         status.enable_state = EffectProcessor::ENABLING;
     }
     return true;
 }
 
-bool EngineEffectChain::disableForGroup(const QString& group) {
-    GroupStatus& status = getGroupStatus(group);
+bool EngineEffectChain::disableForChannel(const ChannelHandle& group) {
+    ChannelStatus& status = getChannelStatus(group);
     if (status.enable_state != EffectProcessor::DISABLED) {
         status.enable_state = EffectProcessor::DISABLING;
     }
     return true;
 }
 
-EngineEffectChain::GroupStatus& EngineEffectChain::getGroupStatus(const QString& group) {
-    for (QLinkedList<GroupStatus>::iterator it = m_groupStatus.begin(),
-                 end = m_groupStatus.end(); it != end; ++it) {
-        GroupStatus& status = *it;
-        if (status.group == group) {
-            return status;
-        }
-    }
-    GroupStatus status(group);
-    m_groupStatus.append(status);
-    return m_groupStatus.last();
+EngineEffectChain::ChannelStatus& EngineEffectChain::getChannelStatus(
+        const ChannelHandle& group) {
+    return m_channelStatus[group];
 }
 
-void EngineEffectChain::process(const QString& group,
+void EngineEffectChain::process(const ChannelHandle& group,
```

Comment:
what's the reason for moving from calling things "groups" to "channels"?  I notice elsewhere you have renamed group -> groupName, but I wonder if the term "group" is obsolete?  Not advocating for a global rename right now, but I think it'll be tough esp for newbies to get what ChannelHandle is now that it's not a simple string.